microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/jaxrs/JaxRsContext.java

```diff
@@ -35,6 +53,8 @@
 	// The value of the @ApplicationPath annotation
 	private String applicationPath;
 
+	private JavaCodeLensContext javaCodeLensContext;

```

Comment:
`private final JavaCodeLensContext javaCodeLensContext;`

Reply:
Final would mean the constructor cannot set it correct?

Reply:
no, final means that constructor initialize it and after that any code cannot change it.